src/docs/asciidoc/jcache/icache.adoc

```diff
@@ -265,16 +251,11 @@ the preferable way to get the name of the instance. Multiple ``CacheManager``s
 created using an equal `java.net.URI` share the same `HazelcastInstance`.
 
 A named scope is applied nearly the same way as the configuration scope.
-Pass the instance name using:
-
-* either the property `HazelcastCachingProvider#HAZELCAST_INSTANCE_NAME`
+You can pass the instance name using the property `HazelcastCachingProvider#HAZELCAST_INSTANCE_NAME`
 (which resolves to `hazelcast.instance.name`) as a mapping inside a `java.util.Properties`
 instance to the `CachingProvider.getCacheManager(uri, classLoader, properties)` call.
-* or use the instance name when specifying the `CacheManager`'s `URI`.
-
-If a valid instance name is provided both as property and as `URI`, then the
-property value takes precedence and is used to resolve the `HazelcastInstance`
-the first time a `CacheManager` is created for the given `URI`.
+When a valid instance name is provided, the property value is used to resolve the `HazelcastInstance`
+the first time a `CacheManager` is created.
```

Comment:
is here a good place to clarify that a client-side `CacheManager` will only look for client `HazelcastInstance` s by that instance name (and respectively the same for member-side instances & cache managers)?

Reply:
I mentioned it under the "Binding to an Existing Hazelcast Instance Object" section, but putting it here, too, to make it more visible.